README.md

```diff
@@ -811,6 +811,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
 ## Monitoring
 
   * [Pingmeter.com](https://pingmeter.com/) - 5 uptime monitors with 10-minute interval. Monitor SSH, HTTP, HTTPS, and any custom TCP ports.
+  * [alerty.ai](https://www.alerty.ai/?utm_source=github&utm_medium=rm&utm_campaign=awesome-developer-first) - Free APM and monitoring for your FE, BE, DB, and more + free agent runs.
```

Comment:
I dont see a commitment to a free tier, this looks more like a early beta trying to attract some users with no clear undertaking to have a free tier:

```
Yes, Alerty is free to use right now. And in the long term Alerty will always be priced for devs. Like seriously. A database on Datadog costs $84 to monitor. Your entire stack on Alerty will cost a small fraction of that.
```

I cant seem to confirm eligibility of this entry since this isnt clear

```
To be eligible, a service must offer a free tier, not just a free trial. The free tier must be for at least a year if it is time-bucketed. 
```

Reply:
Appreciate the comment. I just updated the website section you quoted to include our plan to have a free-forever tier that provides basic monitoring functionality.

Reply:
ok yeah looks better, not just remove all the tracking junk please

Reply:
Done